Makefile.3ds

```diff
@@ -120,7 +120,7 @@ ifeq ($(strip $(ICON)),)
 		endif
 	endif
 else
-	export APP_ICON := $(TOPDIR)/$(ICON)
+	export APP_ICON := $(TOPDIR)/$(APP_ICON)
```

Comment:
I'm sorry, I intended to only have the change in the Makefile not the Makefile.3ds. I was not aware it would also grab the Makefile.3ds which I committed after making the pull request. Sorry!